cleanlab/classification.py

```diff
@@ -206,14 +206,12 @@ class CleanLearning(BaseEstimator):  # Inherits sklearn classifier
       perfectly labeled (you are certain that there are no errors in that class).
 
     find_label_issues_kwargs : dict, optional
-      Keyword arguments to pass into :py:func:`filter.find_label_issues
-      <cleanlab.filter.find_label_issues>`. Particularly useful options include:
+      Keyword arguments to pass into `~cleanlab.filter.find_label_issues`. Particularly useful options include:
```

Comment:
see comment here why this is not ok:  https://github.com/cleanlab/cleanlab/pull/790#issuecomment-1666176986